lib/widgets/autocomplete.dart

```diff
@@ -168,19 +171,22 @@ class ComposeAutocomplete extends AutocompleteField<ComposeAutocompleteQuery, Co
   ComposeContentController get controller => super.controller as ComposeContentController;
 
   @override
-  AutocompleteIntent<ComposeAutocompleteQuery>? autocompleteIntent() => controller.autocompleteIntent();
+  AutocompleteIntent<ComposeAutocompleteQuery>? autocompleteIntent(
+    BuildContext context) => controller.autocompleteIntent();
```

Comment:
These changes are no longer needed, since the query and intent no longer carry the localizations.